samza-test/src/test/java/org/apache/samza/test/processor/TestStreamProcessor.java

```diff
@@ -247,4 +220,70 @@ private void verifyNumMessages(String topic, int expectedNumMessages) {
 
     Assert.assertEquals(count, expectedNumMessages);
   }
+
+  private static class Mocks {
+    CountDownLatch latch;
```

Comment:
minor : prefer this to be `shutdownLatch` to make intent explicit

Reply:
done